asset-template/src/component/index.tsx.hbs

```diff
@@ -14,7 +14,7 @@ const use{{pascalCase assetName}}Pros = (props: Transformed{{pascalCase assetNam
 };
 
 export const {{pascalCase assetName}}Component = (props: Transformed{{pascalCase assetName}}) => {
-  const { children, ...buttonProps } = use{{pascalCase assetName}}Pros(props);
+  const { children, ...buttonProps } = use{{pascalCase assetName}}Props(props);
```

Comment:
Let's use a generic name like `...rest`, for example. The `buttonProps` was kept by mistake on the former generator.